src/main/scala/com/workday/warp/dsl/ExecutionConfig.scala

```diff
@@ -4,6 +4,7 @@ import com.workday.telemetron.math.DistributionLike
 import com.workday.warp.TrialResult
 import com.workday.warp.arbiters.traits.ArbiterLike
 import com.workday.warp.collectors.abstracts.AbstractMeasurementCollector
+import com.workday.warp.collectors.AbstractMeasurementCollectionController
```

Comment:
i think this import is unused

Reply:
Yeah, it's just used in some docs (e.g., line 235). I can remove it if you'd like though

Reply:
i see, can you try `.gradlew checkScalafix` ? 

i am curious about whether scalafix counts that doc usage

Reply:
Ah, it does not. I'll remove this line